pyperf/_runner.py

```diff
@@ -310,6 +310,8 @@ def _process_args_impl(self):
         if args.track_memory:
             if MS_WINDOWS:
                 from pyperf._win_memory import check_tracking_memory
+            elif MAC_OS:
+                from pyperf._psutil_memory import check_tracking_memory
             else:
                 from pyperf._memory import check_tracking_memory
```

Comment:
Maybe pyperf/_memory.py should be a portable API which switchs between the Linux, Windows and macOS implementation. (And _memory.py should be renamed to _linux_memory.py)